src/awesome_map/BoundaryInterceptor.js

```diff
@@ -225,6 +243,37 @@ define(function(require) {
             var eventType = event.type;
             var originalState;
 
+
+            // Boundary Detection
+            // @TODO: It seems that this event can be triggered by inertial events generated
```

Comment:
For this, one, I believe that we probably want the event to fire based on inertia.